### DISCUSSION TITLE
false positive event suggestions

### DISCUSSION THREAD

**Discussion by jsbowling42** [blakeblackshear/frigate#5140] [Ask A Question] [answered]
i got an event for this truck, saying that it was in my driveway zone( orange zone ). any thoughts on a way to keep this from happening?
![image](https://user-images.githubusercontent.com/35424582/212970292-811171b7-ea30-4f07-87fc-1b85076002a0.png)

ive created a test camera feed with the recording of the truck going by and while i cant see the bounding box enter the zone i guess it does as frigate creates an event for it too.

**Comment by NickM-27** [comment, marked as answer]
the debug view only shows at 5 fps and isn't a direct representation of what originally happened. I am not sure there's much that can be done to prevent this on the user side since it is a true positive. We are planning to work on tracking and other object improvements in the next release so this should hopefully improve.
